migration/migrationsUp.sql

```diff
@@ -0,0 +1,12 @@
+-- CREATE TABLE "user" ----------------------------------------
+CREATE TABLE IF NOT EXISTS "user" (
+    "id" UUid DEFAULT gen_random_uuid() NOT NULL,
+    "first_name" VARCHAR(255) NOT NULL,
+    "last_name" VARCHAR(255),
+    "email" VARCHAR(255) NOT NULL,
```

Comment:
we can use `UNIQUE` here and get rid of `CONSTRAIN`, right? Or you wanted to declare it more explicitly?

Reply:
There is way to name explicitly constrain. That's why I did it like this